pinot-spi/src/main/java/org/apache/pinot/spi/env/CommonsConfigurationUtils.java

```diff
@@ -42,29 +44,20 @@
  * Provide utility functions to manipulate Apache Commons {@link Configuration} instances.
  */
 public class CommonsConfigurationUtils {
+
+  private static final String VERSIONED_CONFIG_SEPARATOR = " = ";
```

Comment:
is the white space required?

Reply:
This is the same value of `DEFAULT_SEPARATOR` in [PropertiesConfiguration.java](https://github.com/apache/commons-configuration/blob/15e6ec62f1b309bab0bffd139a4790564dc3ca73/src/main/java/org/apache/commons/configuration2/PropertiesConfiguration.java#L1005). We are using the same because in VersionedPropertyReader we are using the [getPropertySeparator](https://github.com/apache/pinot/pull/12440/files#diff-7cfc074e3db46c25491eba3d6df78847cecbb093b4780dbe6a1b8875ffdc3555R42) to split the string into key and value.